src/main/java/edu/indiana/dlib/amppd/controller/BatchController.java

```diff
@@ -117,11 +118,14 @@ public String runPreprocessingOnFilesWithoutMediaInfo(@RequestParam(value = "pri
 		    log.info("serveFile Initiated");
```

Comment:
can you change the endpoint to /static/{fileName:.+}, since this is for static web content? On a hindsight, BatchController may not be the best place to put this method, since it's for generic static content not limited to batch related files. However, we can leave it as is for now.

A curious question: have you tried getting the file without adding any endpoint? Based on this article, it seems that Spring Boot will take care of serving static content as long as the file is within one of those conventional locations: https://spring.io/blog/2013/12/19/serving-static-web-content-with-spring-boot

Reply:
Yes Ying Without writing Java Code directly accessing  /rest/AmpBatchIngestTemplate.csv endpoint is Working can I Remove Controller Method and Test File